ha/fault_tolerance/fault_tolerance_driver.py

```diff
@@ -40,6 +41,7 @@ def __init__(self):
         ConfigManager.init("fault_tolerance")
         self.node_fault_monitor = HealthStatusMonitor()
         self.cluster_stop_monitor = ClusterStopMonitor()
+        ConftStoreSearch()
```

Comment:
Discussed with Mariyappan for this! It is introduced here so that in system health, setach API call will be used to get cvg ids.
It is temporary code for now. Will be removed in next ticket.